### PR TITLE
Regenerated intellisense xml from winforms triple slash comments

### PR DESCRIPTION
**Pull request by AdamYoblick**
Fixes https://github.com/dotnet/winforms/issues/4163

I followed the steps described at https://github.com/carlossanlop/DocsPortingTool.

The command line I ran was `.\DocsPortingTool\bin\Debug\net5.0\DocsPortingTool.exe -Docs E:\src\repos\github\dotnet-api-docs\xml -TripleSlash E:\src\repos\github\winforms\artifacts\bin -IncludedAssemblies Accessibility,Microsoft.VisualBasic.Forms,System.Windows.Forms.Design,System.Windows.Forms.Primitives,System.Windows.Forms -Save true`

Please let me know if you have any concerns 😄

### CODE REVIEW

**Comment by AdamYoblick** [comment]
I cant seem to add reviewers... @carlossanlop @RussKie @JeremyKuhne Please take a look

**Comment by carlossanlop** [comment]
> _I cant seem to add reviewers_

I added you to the list of people with write permissions for this repo, @AdamYoblick. Please accept the invite. Next time you'll be able to add reviewers.

File: xml/System.Windows.Forms.VisualStyles/FontProperty.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms.VisualStyles/FontProperty.xml:49]: So this is a tiny bug in the tool, in which if it does not find a `<remarks>` item in the dotnet-api-docs xml, it will add it with the default boilerplate text.

It's harmless, but if you prefer to remove this (and all other instances) so the PR is smaller, feel free to do it. Totally optional.

I will get it fixed in the tool as soon as I get a chance.

File: xml/System.Windows.Forms/AccessibleEvents.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/AccessibleEvents.xml:775]: Is this a remark only relevant for developers reading the source code? `TrackPopupMenu()` seems to be [a Windows method](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-trackpopupmenu), that's why I ask.

If you think that is the case, we can remove it:

```suggestion
        <remarks>To be added.</remarks>
```

File: xml/System.Windows.Forms/AccessibleEvents.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/AccessibleEvents.xml:807]: Same. Feel free to apply the suggestion if this is not something that matters to people reading the API description in MS Docs:

```suggestion
        <remarks>To be added.</remarks>
```

File: xml/System.Windows.Forms/BindingContext.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/BindingContext.xml:212]: Here and elsewhere - Since the `<remarks>` information does not show up in Intellisense in Visual Studio, I think we should move this text to the `<summary>`, although I am not sure what is the best guidance for mentioning obsoletion. What would you suggest, @gewarren?

File: xml/System.Windows.Forms/ListView.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ListView.xml:2488]: I don't see `GroupIcon` reflected in the DocId of this API. I see `ImageList`. Is this description accurate?

File: xml/System.Windows.Forms/ListView.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ListView.xml:2501]: Merging 4 empty lines into one:
```suggestion
             
```

File: xml/System.Windows.Forms/ListView.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ListView.xml:4151]: Missing parameter description here and in the next API below. Can you please add it?

File: xml/System.Windows.Forms/ListViewGroup.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ListViewGroup.xml:500]: Here and in the next API, can you add the missing `<value>`?

File: xml/System.Windows.Forms/ListViewGroupEventArgs.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ListViewGroupEventArgs.xml:59]: Missing value.

File: xml/System.Windows.Forms/ScrollBar.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/ScrollBar.xml:442]:
```suggestion
        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.ScrollBar" /> instance.
             The <see cref="T:System.Windows.Forms.AccessibleObject" /> instance returned by this method supports ControlType UIA property.</summary>
        <returns><see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.ScrollBar" /> instance.</returns>
```

File: xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml:84]: This API and the few below are missing a summary and param descriptions. Can you please add them?

File: xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml:119]:
```suggestion
          <paramref name="item" /> is already a part of a different collection.

-or-

This collection is currently bound to a task dialog.</exception>
```

File: xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml:177]:
```suggestion
          <paramref name="item" /> is already a part of a different collection.
          
-or-

This collection is currently bound to a task dialog.</exception>
```

File: xml/System.Windows.Forms/TextFormatFlags.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextFormatFlags.xml:184]: Since remarks do not show up in intellisense in VS, This should be mentioned in the summary instead.

File: xml/System.Windows.Forms/TextFormatFlags.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextFormatFlags.xml:284]: Same as above. Please move to summary.

File: xml/System.Windows.Forms/TextFormatFlags.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextFormatFlags.xml:643]: ditto

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:319]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:392]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:673]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:712]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:874]: Maybe some of this information could be moved to the remarks. We probably don't need all this information to be clustered in the return value, we only need the essential description.

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:1188]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/TextRenderer.xml:1277]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><see cref="F:System.Windows.Forms.TextFormatFlags.ModifyString" /> is set.</exception>
```

File: xml/System.Windows.Forms/WebBrowserBase.xml
Comment (carlossanlop) [inline xml/System.Windows.Forms/WebBrowserBase.xml:2623]: Is this information that the average MS Docs user would be interested in, or is this aimed to the maintainers of the source code?

```suggestion
Certain messages are forwarder directly to the ActiveX control, others are first processed by the wndproc of Control.
```

**Review by carlossanlop — COMMENTED** [review]
Thank you for submitting, @AdamYoblick . I left some suggestions for you to consider.

**Review by RussKie — APPROVED** [review]
LGTM 👍

File: xml/System.Windows.Forms/ListView.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ListView.xml:4152]:
```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.ListView.GroupCollapsedStateChanged" /> event.</summary>
```

See https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Event-related.

File: xml/System.Windows.Forms/ListView.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ListView.xml:4176]:
```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.ListView.GroupTaskLinkClick" /> event.</summary>
```

File: xml/System.Windows.Forms/ListViewGroup.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ListViewGroup.xml:499]:
```suggestion
        <summary>Gets or sets the text displayed in the group subtitle.</summary>
```

File: xml/System.Windows.Forms/ListViewGroup.xml
Comment (gewarren) [inline xml/System.Windows.Forms/ListViewGroup.xml:609]:
```suggestion
        <summary>Gets or sets the name of the task link displayed in the group header.</summary>
```

File: xml/System.Windows.Forms/TextFormatFlags.xml
Comment (gewarren) [reply xml/System.Windows.Forms/TextFormatFlags.xml:184]: I don't think remarks for enum fields show up even in docs.

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:89]:
```suggestion
        <param name="font">The font to apply to the drawn text.</param>
```

Types are shown automatically.

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:90]:
```suggestion
        <param name="pt">The point that represents the upper-left corner of the drawn text.</param>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:91]:
```suggestion
        <param name="foreColor">The color to apply to the drawn text.</param>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:874]:
```suggestion
        <returns>The size, in pixels, of text drawn on a single line with the specified font.</returns>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:947]:
```suggestion
        <returns>The size, in pixels, of <paramref name="text" /> drawn with the specified <paramref name="font" /> in the specified device context.</returns>
```

File: xml/System.Windows.Forms/TextRenderer.xml
Comment (gewarren) [inline xml/System.Windows.Forms/TextRenderer.xml:1271]:
```suggestion
        <param name="flags">A bitwise combination of the enumeration values that specifies the formatting instructions to apply to the measured text.</param>
```

File: xml/System.Windows.Forms.VisualStyles/FontProperty.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms.VisualStyles/FontProperty.xml:49]: I see you fixed this in the tool. I'll regenerate these files. :)

Comment (AdamYoblick) [reply xml/System.Windows.Forms.VisualStyles/FontProperty.xml]: @carlossanlop I used the latest version of the tool, but it seemed to add lots of remarks now regarding explicit interface implementations. Is this intended?

File: xml/System.Windows.Forms/AccessibleEvents.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms/AccessibleEvents.xml:746]: @RussKie You added this comment a while back, should we leave it as is?

File: xml/System.Windows.Forms/BindingContext.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms/BindingContext.xml:212]: @RussKie There are several methods in `BindingContext.cs` that are called out as obsolete in the remarks. Should we decorate them with `[Obsolete]` so the docs get generated correctly?

File: xml/System.Windows.Forms/BindingContext.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms/BindingContext.xml:206]: @carlossanlop @gewarren I have a question about the tool that ports triple slash comments to the doc xml. I'm looking at our BindingContext.cs and I see this for the summary:

```
/// <summary>
///  Adds the listManager to the collection. An ArgumentNullException is thrown if this
///  listManager is null. An exception is thrown if a listManager to the same target
///  and Property as an existing listManager or if the listManager's column isn't a
///  valid column given this DataSource.Table's schema.
///  Fires the CollectionChangedEvent.
/// </summary>
```

But in BindingContext.xml, the summary does not match the code. Did I miss a flag on the command line or something?

File: xml/System.Windows.Forms/DrawItemEventArgs.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms/DrawItemEventArgs.xml:601]: @carlossanlop This also doesn't match what's in our code. The `GetHdc()` method in `DrawItemEventArgs.cs` has literally no XML docs, so the tool should have cleared these out?

File: xml/System.Windows.Forms/ListView.xml
Comment (AdamYoblick) [reply xml/System.Windows.Forms/ListView.xml:2501]: The remarks block on this method looks like this:

```
        /// <remarks>
        ///  <para>
        ///   The <see cref="GroupImageList"/> property allows you to specify an <see cref="ImageList"/> object that
        ///   contains icons to use when displaying groups. The <see cref="ListView"/> control can accept any graphics
        ///   format that the <see cref="ImageList"/> control supports when displaying icons. The <see cref="ListView"/>
        ///   control is not limited to .ico files. Once an <see cref="ImageList"/> is assigned to the <see cref="GroupImageList"/>
        ///   property, you can set the <see cref="ListViewGroup.TitleImageIndex"/> property of each <see cref="ListViewGroup"/>
        ///   in the <see cref="ListView"/> control to the index position of the appropriate image in the <see cref="ImageList"/>.
        ///   The size of the icons for the <see cref="GroupImageList"/> is specified by the <see cref="ImageList.ImageSize"/> property.
        ///  </para>
        /// </remarks>
```

I'm not sure where these empty lines are coming from

**Comment by AdamYoblick** [comment]
I'm not going to lie, this task blew up in scope. I just wanted to make sure the intellisense docs were up to date since we have a bunch of debt in this area (i.e. completely missing API). I think we should make a separate task to "fix up" comments where appropriate.

File: xml/System.Windows.Forms.VisualStyles/FontProperty.xml
Comment (carlossanlop) [reply xml/System.Windows.Forms.VisualStyles/FontProperty.xml]: It wasn't intended but it doesn't hurt.
I just fixed that bug in the newest version. It should not happen again.

**Pull request merged**
